Add possibility for Host packages to ask Buildroot to build libusb.
This way user is not obliged to install libusb devt packages on its distrib.

Signed-off-by: Julien Boibessot <julien.boibessot@armadeus.com>

Index: buildroot/package/libusb/libusb.mk
===================================================================
--- buildroot.orig/package/libusb/libusb.mk	2012-01-30 14:52:34.000000000 +0100
+++ buildroot/package/libusb/libusb.mk	2012-01-30 14:52:56.000000000 +0100
@@ -11,3 +11,4 @@
 LIBUSB_INSTALL_TARGET = YES
 
 $(eval $(call AUTOTARGETS,package,libusb))
+$(eval $(call AUTOTARGETS,package,libusb,host))
Index: buildroot/package/libusb-compat/libusb-compat.mk
===================================================================
--- buildroot.orig/package/libusb-compat/libusb-compat.mk	2012-01-30 15:21:20.000000000 +0100
+++ buildroot/package/libusb-compat/libusb-compat.mk	2012-01-30 15:40:32.000000000 +0100
@@ -7,6 +7,7 @@
 LIBUSB_COMPAT_SOURCE = libusb-compat-$(LIBUSB_COMPAT_VERSION).tar.bz2
 LIBUSB_COMPAT_SITE = http://$(BR2_SOURCEFORGE_MIRROR).dl.sourceforge.net/project/libusb/libusb-compat-0.1/libusb-compat-$(LIBUSB_COMPAT_VERSION)
 LIBUSB_COMPAT_DEPENDENCIES = host-pkg-config libusb
+HOST_LIBUSB_COMPAT_DEPENDENCIES = host-pkg-config host-libusb
 LIBUSB_COMPAT_INSTALL_STAGING = YES
 LIBUSB_COMPAT_INSTALL_TARGET = YES
 
@@ -19,3 +20,4 @@
 LIBUSB_COMPAT_POST_INSTALL_STAGING_HOOKS+=LIBUSB_COMPAT_FIXUP_CONFIG
 
 $(eval $(call AUTOTARGETS,package,libusb-compat))
+$(eval $(call AUTOTARGETS,package,libusb-compat,host))
